torch/csrc/distributed/rpc/init.cpp

```diff
@@ -626,15 +630,22 @@ If the future completes with an error, an exception is thrown.
 
   module.def(
       "_set_rpc_timeout",
-      [](const std::chrono::milliseconds& rpcTimeout) {
+      [](const float rpcTimeoutSeconds) {
+        const auto secToMsConversion = 1000;
```

Comment:
Shall we use one const for `secToMsConversion` and `msToSecDiv`? It can be a float as we are doing the cast anyway?

Reply:
Changed it to be a single const